plugins/modules/content_export_version.py

```diff
@@ -0,0 +1,169 @@
+      - Fails if any of the repositories belonging to this version are unexportable.
+    required: false
+    type: bool
+  incremental:
+    description:
+      - Export only the content that has changed since the last export.
+    required: false
+    type: bool
+  from_history_id:
+    description:
+      - Export history identifier used for incremental export. If not provided the most recent export history will be used.
+    required: false
+    type: str
+extends_documentation_fragment:
+  - theforeman.foreman.foreman
+  - theforeman.foreman.foreman.organization
+'''
+
+EXAMPLES = '''
+- name: "Export content view version (full)"
+  content_export_version:
+    content_view: RHEL8
+    content_view_version: '1.0'
+    username: "admin"
+    password: "changeme"
+    server_url: "https://foreman.example.com"
+    organization: "Default Organization"
+    destination_server: "airgapped.example.com"
+
+- name: "Export content view version (full) in chunks of 10 GB"
+  content_export_version:
+    content_view: RHEL8
+    content_view_version: '1.0'
+    username: "admin"
+    password: "changeme"
+    server_url: "https://foreman.example.com"
+    organization: "Default Organization"
+    destination_server: "airgapped.example.com"
+    chunk_size_gb: 10
+
+- name: "Export content view version (full) and fail if any repos are unexportable"
+  content_export_version:
+    content_view: RHEL8
+    content_view_version: '1.0'
+    username: "admin"
+    password: "changeme"
+    server_url: "https://foreman.example.com"
+    organization: "Default Organization"
+    destination_server: "airgapped.example.com"
+    fail_on_missing_content: true
+
+- name: "Export content view version (incremental) since the most recent export"
+  content_export_version:
+      content_view: RHEL8
+      content_view_version: '1.0'
+      username: "admin"
+      password: "changeme"
+      server_url: "https://foreman.example.com"
+      organization: "Default Organization"
+      destination_server: "airgapped.example.com"
+      incremental: true
+
+- name: "Export content view version (incremental) since a specific export"
+  content_export_version:
+      content_view: RHEL8
+      content_view_version: '1.0'
+      username: "admin"
+      password: "changeme"
+      server_url: "https://foreman.example.com"
+      organization: "Default Organization"
+      destination_server: "airgapped.example.com"
+      incremental: true
+      from_history_id: 12345
+'''
+
+from ansible_collections.theforeman.foreman.plugins.module_utils.foreman_helper import KatelloAnsibleModule, _flatten_entity
+
+
+class KatelloContentExportModule(KatelloAnsibleModule):
+    pass
+
+
+def main():
+    module = KatelloContentExportModule(
+        foreman_spec=dict(
+            content_view_version=dict(type='entity', scope=['content_view'], search_by='version', flat_name='id', required=False),
+            content_view=dict(type='entity', scope=['organization'], required=False),
```

Comment:
@jeremylenz under which circumstances are those not required?

https://github.com/Katello/katello/blob/master/app/controllers/katello/api/v2/content_exports_controller.rb#L31 lists the `id` as required, and we can only get the `id` if we know the name of the CV and the "name/version" of the version, so I'd mark both as required, right?